src/app-host/live-reload-client.js

```diff
@@ -53,15 +53,9 @@ module.exports.start = function (sock) {
      * @returns {boolean} Whether the URL points to the modified file from the server.
      */
     function urlMatchesPath(url, fileRelativePath) {
-        var isMatch = false;
-
-        localUrlPrefixes.find(function (prefix) {
-            isMatch = prefix + fileRelativePath === url;
-
-            return isMatch;
+        return localUrlPrefixes.some(function (prefix) {
+            return (prefix + fileRelativePath === url);
```

Comment:
these parenthesis do not make much sense. they are either not needed or, in case the precedence needs to be emphasized, the concatenation should be within the parenthesis, like this: `return (prefix + fileRelativePath) === url`.